.github/pull_request_template.md

```diff
@@ -1,32 +1,42 @@
 ## What changes are proposed in this pull request?
  
-(Please fill in changes proposed in this fix)
```

Comment:
Let's undo changes to this file - it's maybe a bit confusing, but you can edit your PR description (as you've done) without editing this template file :)